src/ui/component/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react'
+import type { FC } from 'react'
+import classNames from 'classnames'
+import * as RTooltip from '@radix-ui/react-tooltip'
+import './tooltip.scss'
+
+type TooltipProps = {
+  content: JSX.Element
+  trigger: JSX.Element
+  contentClassName?: string
+  triggerClassName?: string
+  side?: 'left' | 'right' | 'top' | 'bottom'
+  sideOffset?: number
+  container?: HTMLElement
+}
```

Comment:
Here I would add a prop to propose to show the tooltip instantly as proposed in radix: `delayDuration` .
When I tried the tooltip I was thinking that it wasn't working because of the small delay ^^

Reply:
Definitely agree. `delayDuration` props has been added, with a shorter default delay (400ms).